config/routes.rb

```diff
@@ -12,8 +12,12 @@
     get "/accessibility-statement", to: "accessibility_statement#show"
 
     # Redirect for deleted question and page (301 is default)
-    get "/urgent-medical-help", to: redirect("/need-help-with")
-    get "/get-help-from-nhs", to: redirect("/need-help-with")
+    get "/urgent-medical-help", to: redirect("/where-live")
+    get "/get-help-from-nhs", to: redirect("/where-live")
```

Comment:
Could the first question be defined as an application wide constant, which can then be referenced elsewhere, e.g. https://github.com/alphagov/govuk-coronavirus-find-support/pull/175/commits/ba03935f826cfa74c3bdaf62ae6ea753369d7ce7#diff-55c5b7aecfb519d0e4880eaf2788eb6eR40 and all the tests?  It will make it easier to change the first question if we need to do this again.

Reply:
yeah I think this is a good shout. For the sake of getting this released can we add it as a new story though? Kind of aware that this has taken me a while to get in and it'd be good to get this functionality added.

Reply:
Yes, that's absolutely fine with me 👍